src/GitVersionCore/Configuration/Init/SetConfig/SetBranchIncrementMode.cs

```diff
@@ -1,37 +1,41 @@
 using System.Collections.Generic;
 using GitVersion.Configuration.Init.Wizard;
 using GitVersion.VersioningModes;
-using GitVersion.Common;
 using GitVersion.Logging;
 
 namespace GitVersion.Configuration.Init.SetConfig
 {
     public class SetBranchIncrementMode : ConfigInitWizardStep
     {
-        private readonly string name;
-        private readonly BranchConfig branchConfig;
+        private string name;
+        private BranchConfig branchConfig;
 
-        public SetBranchIncrementMode(string name, BranchConfig branchConfig, IConsole console, IFileSystem fileSystem, ILog log)
-            : base(console, fileSystem, log)
+        public SetBranchIncrementMode(IConsole console, IFileSystem fileSystem, ILog log, IConfigInitStepFactory stepFactory) : base(console, fileSystem, log, stepFactory)
         {
-            this.name = name;
-            this.branchConfig = branchConfig;
+        }
+
+        public SetBranchIncrementMode WithData(string _name, BranchConfig _branchConfig)
+        {
+            branchConfig = _branchConfig;
+            name = _name;
```

Comment:
Use `this.name` to avoid having to prefix with `_`.

Reply:
adjusted field names, by removing underscore